### PR TITLE
Add `null` to `json`

### PR DESCRIPTION
**Pull request by apalm**
Resolves https://github.com/jsverify/jsverify/issues/170.

### CODE REVIEW

File: lib/json.js [hunk 1]
```diff
@@ -11,13 +11,22 @@ var shrink = require("./shrink.js");
 var string = require("./string.js");
 var utils = require("./utils.js");
 
+var nulll = primitive.constant(null);
```
Comment (apalm) [inline lib/json.js]: I wasn't sure if this is better than e.g.
```js
var nulll = arbitraryBless({
  generator: generator.constant(null),
  shrink: shrink.noop,
  show: show.def,
});
```

Also, suggestions for a better name than `null` welcome 😅

Comment (StoneCypher) [inline lib/json.js]: `null_constant`

Comment (phadej) [inline lib/json.js]: `nullArb`.

@StoneCypher I don't use `snake_case` in the code.

**Comment by phadej** [comment]
thanks

File: lib/json.js (hunk 1 above)
Comment (StoneCypher) [inline lib/json.js]: i was joking

**Pull request merged**